deeplake/api/dataset.py

```diff
@@ -833,6 +833,10 @@ def delete(
                 logger.info(f"{path} dataset deleted successfully.")
         except Exception as e:
             if force:
+                if get_path_type(path) == "local" and not os.path.isdir(path):
```

Comment:
is it the only possible case? What happens if I specify incorrect s3 path or gcs?

Reply:
will update the PR to raise an error if the specified path is empty